interface/patient_file/history/encounters.php

```diff
@@ -33,6 +33,9 @@
 
 $is_group = ($attendant_type == 'gid') ? true : false;
 
+if (isset($_GET['pid']) && $_GET['pid'] != $_SESSION['pid']) {
+    setpid($_GET['pid']);
+}
```

Comment:
almost there @sjpadgett!

getting an Unknown tab when switching between people when clicking on the Insurance button from the billing manager
![Screenshot from 2024-10-01 16-18-31](https://github.com/user-attachments/assets/d7e645cc-bbea-448d-8f52-97880940871c)

Encounter button loads fine:
![Screenshot from 2024-10-01 16-15-53](https://github.com/user-attachments/assets/92d57dab-76f7-4dae-b872-1b19d6c3c0d5)

Insurance button issue:
![Screenshot from 2024-10-01 16-19-30](https://github.com/user-attachments/assets/fbd2d50f-9a74-4830-be75-64e17fc7d66e)

Reply:
I switched database to one with modules and api enabled. I'm getting
```
[01-Oct-2024 16:29:23 America/New_York] PHP Fatal error:  Uncaught Error: Call to undefined function setpid() in C:\xampp\htdocs\openemr\interface\patient_file\history\encounters.php:37
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\openemr\interface\patient_file\history\encounters.php on line 37
[01-Oct-2024 16:29:24 America/New_York] [2024-10-01T16:29:24.986858-04:00] OpenEMR.DEBUG: HttpRestRouteHandler::dispatch() start request {"resource":"","method":"GET","user":"","role":"","client":"","apiType":"oemr","route":"/api/patient/9353b8f5-0a87-4e2a-afd4-25341fdb0fbc/insurance","queryParams":[]} []
[01-Oct-2024 16:29:24 America/New_York] [2024-10-01T16:29:24.991857-04:00] OpenEMR.DEBUG: HttpRestParsedRoute->__construct() matched {"routePath":"GET /api/patient/:puuid/insurance","requestPath":"/api/patient/9353b8f5-0a87-4e2a-afd4-25341fdb0fbc/insurance","method":"GET","routeParams":["9353b8f5-0a87-4e2a-afd4-25341fdb0fbc"],"resource":"insurance","operation":""} []
[01-Oct-2024 16:29:24 America/New_York] [2024-10-01T16:29:24.993213-04:00] OpenEMR.DEBUG: HttpRestRouteHandler->dispatch() dispatching route {"route":"GET /api/patient/:puuid/insurance"} []
[01-Oct-2024 16:29:25 America/New_York] [2024-10-01T16:29:25.034661-04:00] OpenEMR.DEBUG: RestControllerHelper::handleFhirProcessingResult() Records found {"count":"3"} []

```

However, behavior is not consistent! Weird stuff. Man! Look what you did to me!:)
I'll look at a little later. I need to figure out why batch notifications aren't working from command line so I can sleep tonight.:)

Reply:
@stephenwaite This is working for me now. Let me know if you see any issues

Reply:
ok, thanks, that fix makes sense, will check it out